web/controllers/homepage_controller.ex

```diff
@@ -72,9 +73,9 @@ defmodule App.HomepageController do
 
     all_resources =
       articles ++ resources
-      |> Enum.map(&(get_all_tags(&1)))
-      |> filter_tags(audience_filter, content_filter)
-      |> Enum.sort(&(&1[:id] <= &2[:id]))
+      |> Enum.map(&(Resources.get_all_tags(&1)))
```

Comment:
Not a big deal, but I have a slight preference for the syntax:

```elixir
|> Enum.map(&Resources.get_all_tags/1)
```

Less brackets + easier to read (imo) + write